src/test/java/com/eleks/academy/whoami/core/impl/PersistentPlayerTest.java

```diff
@@ -0,0 +1,22 @@
+package com.eleks.academy.whoami.core.impl;
+
+import org.junit.jupiter.api.Test;
+
+import java.util.concurrent.Future;
+
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.junit.jupiter.api.Assertions.assertThrows;
```

Comment:
⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.imports.CustomImportOrderCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Import statement for 'org.junit.jupiter.api.Assertions.assertThrows' is in the wrong order. Should be in the 'STATIC' group, expecting not assigned imports on this line.